README.rst

```diff
@@ -542,6 +542,23 @@ It sets the number of seconds the notification contents should be stored in the
 If you want cached notification content to never expire, then set it to ``None``.
 Set it to ``0`` if you don't want to store notification contents in cache at all.
 
+
+Exceptions
+----------
+
+NotificationRenderException
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    openwisp_notifications.utils.NotificationRenderException
+
+Raised when notification properties(``email`` or ``message``) cannot be rendered from
```

Comment:
defining an exception in utils.py is very unusual, exceptions.py is more common: `openwisp_notifications.exceptions.NotificationRenderException`, look for "Rule of least astonishment".